.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,220 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: Close "Need More Info" Issues
+      frequencies:
+      - hourly:
+          hour: 3
+      filters:
+      - isOpen
+      - isIssue
+      - hasLabel:
+          label: Need More Info
+      - noActivitySince:
+          days: 10
+      actions:
+      - closeIssue
+      - addReply:
+          reply: "Closing this issue as we've seen no reply to the request for more information. If you are able to get the requested information, please add it to the issue and we will retriage it. "
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      - isPullRequest
+      - labelAdded:
+          label: auto-merge
+      - or:
+        - activitySenderHasPermission:
+            permission: Admin
+        - isActivitySender:
+            user: dotnet-bot
+            issueAuthor: False
+      then:
+      - approvePullRequest:
+          comment: Auto-approval
+      description: Auto-approve auto-merge PRs
+    - if:
+      - payloadType: Pull_Request
+      - isPullRequest
+      - isActivitySender:
+          user: dotnet-maestro[bot]
+          issueAuthor: False
+      - isAction:
+          action: Opened
+      - not:
+          bodyContains:
+            pattern: Updates sdk.version
+            isRegex: False
+      then:
+      - approvePullRequest:
+          comment: Auto-approve
+      description: Auto-approve maestro PRs
+    - if:
+      - payloadType: Pull_Request
+      - isPullRequest
+      - or:
+        - targetsBranch:
+            branch: main
+        - targetsBranch:
+            branch: main-vs-deps
+      - isAction:
+          action: Null

```

Comment:
I think this condition is lost. This policy should run when the PR is merged, so the `merged` should be the action here.

Reply:
See Line 146 in the fabricbot.json for the relevant task.

Reply:
This was discussed offline, and it was caused by `merged` actions that are no longer supported. The config was updated to use the `closed` action with the condition of `isMerged`.